onnx/defs/logical/defs.cc

```diff
@@ -138,4 +138,68 @@ ONNX_OPERATOR_SET_SCHEMA(
             "Constrains input/output to boolean tensors.")
         .TypeAndShapeInferenceFunction(unaryLogicalOpInference));
 
+static const char* LeftShift_ver10_doc = R"DOC(
```

Comment:
I think it's better to have on op to current both case (right and left).

Reply:
No problem.